daemon/rpc_test.go

```diff
@@ -543,6 +551,11 @@ func serversList() core.Servers {
 		},
 	}
 
+	// if at least one record is not valid - reject whole list, asuming something wrong is with whole list
```

Comment:
[small] Also here `asuming`

Reply:
And also the order, here isn't quite right - `something is wrong`